bumble/hci.py

```diff
@@ -557,6 +557,13 @@ def phy_list_to_bits(phys):
 HCI_LE_SET_DEFAULT_SUBRATE_COMMAND                                       = hci_command_op_code(0x08, 0x007D)
 HCI_LE_SUBRATE_REQUEST_COMMAND                                           = hci_command_op_code(0x08, 0x007E)
 
+# Vendor specific HCI commands
+HCI_VSC_TX_POWER_HANDLE_TYPE_ADV = 0x00
```

Comment:
Those 3 constants should be class members of the `HCI_VSC_Write_Tx_Power_Level_Command` class I think.